tests/lib/rules/jsx-curly-brace-presence.js

```diff
@@ -469,13 +469,16 @@ ruleTester.run('jsx-curly-brace-presence', rule, {
       features: ['no-ts'],
       options: ['never'],
     },
+    // legit as this single template literal might be used for type casting
     {
-      code: '<App label={`${label}${suffix}`} />',
-      options: [{ props: 'never' }],
+      code: '<App label={`${label}`} />',
+      output: '<App label={label} />',
+      options: ['never'],
     },
     {
-      code: '<App>{`${label}${suffix}`}</App>',
-      options: [{ children: 'never' }],
+      code: '<App>{`${label}`}</App>',
+      output: '<App>{label}</App>',
```

Comment:
any particular reason why we keep `output` ?

Reply:
no, actually this line should be removed